README.md

```diff
@@ -162,4 +177,142 @@ $ oc get image | grep onload
 ```
 Remove any outstanding manually with `oc delete image`. (Not providing any automated invocation to prevent removal of the false-positive images.)
 
+## Deploying artifacts into an airgapped cluster
+
+### Images needed
+
+This is a list of the images that are currently needed as reported by podman.
+```
+$ sudo podman images --format "table {{.Repository}} {{.Tag}}"
+REPOSITORY                                                                                           TAG
+default-route-openshift-image-registry.apps.test.kube.test/default/onload-sfnettest                  sfnettest-1.6.0-rc1
+default-route-openshift-image-registry.apps.test.kube.test/sfptpd/sfptpd                             git-ab881b3
+default-route-openshift-image-registry.apps.test.kube.test/onload-clusterlocal/onload-module         git27b3826-4.18.0-372.49.1.el8_6.x86_64
+default-route-openshift-image-registry.apps.test.kube.test/onload-clusterlocal/onload-device-plugin  latest
+default-route-openshift-image-registry.apps.test.kube.test/onload-clusterlocal/onload-user           git27b3826
+default-route-openshift-image-registry.apps.test.kube.test/openshift-kmm/sfc-module                  git27b3826-4.18.0-372.49.1.el8_6.x86_64
+```
+
+### Getting the images
+
+The images can either be built using podman directly (untested) or using a
+non-airgapped cluster.
+
+
+#### Pulling images from a cluster
```

Comment:
OpenShift sysadmins will be highly proficient in image handling so will not require an introduction to podman. This section would be better served by a ready-to-go script covering all the necessary images.

Reply:
I think I'm going to leave this as is for the time being since this document is useful to members of our team who aren't proficient with image handling.

Reply:
How about moving the `podman` sections into a separate README file? The top-level one becomes overwhelming. I bet we should keep it short and friendly!

If you think the main consumers are going to be our team members (who are highly skilled! 😜), how about moving these bits into the internal knowledge base like Confluence?